doc/manual/src/protocols/derivation-aterm.md

```diff
@@ -0,0 +1,9 @@
+# Derivation on-disk "ATerm" format
+
+For historical reasons, [derivations](@docroot@/glossary.md#gloss-store-derivation) are stored on-disk in [ATerm](https://homepages.cwi.nl/~daybuild/daily-books/technology/aterm-guide/aterm-guide.html) format.
+
+Derivations are serialised in the following format:
+
+```
+Derive(...)
```

Comment:
This surely needs a bit more detail

Reply:
The next PR will had some more detail. I just wanted to put something here to have a clear diff.